src/python/WMCore/MicroService/Service/RestApiHub.py

```diff
@@ -32,4 +32,5 @@ def __init__(self, app, config, mount):
         cherrypy.log("MicroService entire configuration:\n%s" % Configuration.getInstance())
         cherrypy.log("MicroService REST configuration subset:\n%s" % config)
 
-        self._add({"data": Data(app, self, config, mount)})
+        self._add({"status": Data(app, self, config, mount),
```

Comment:
I have the impression this is what causes the duplicate records in the logging. Still trying to understand it though...